pkg/chunk/aws/dynamodb_storage_client.go

```diff
@@ -462,7 +462,7 @@ type chunksPlusError struct {
 	err    error
 }
 
-// GetChunks implements chunk.ObjectClient.
+// GetChunks implements chunk.ChunkClient.
```

Comment:
The naming `chunk.ChunkClient` is references in comments across several files, but it should be `chunk.Client`.

Reply:
👍